docs/data/routes/guides/multisite-guide/content-patterns/en.md

```diff
@@ -0,0 +1,35 @@
+---
+name: content-patterns
+routeTemplate: ./data/component-templates/guide.yml
+title: Content Patterns
+---
+
+> ⚠️ **Content Wanted** - What other multisite-specific content requirements do you come across?
```

Comment:
We actually have a items those are stored globally (says outside the site roots) and represents pages with presentation, local datasources and etc. This pages are used by other sites. Let me know if you are interested in to cover this case. I will share more details how we handle that.

Reply:
O cool! We have a code sample for how to share datasources across different SXA Tenants (even if it's a mix of JSS and MVC Tenants), but it sounds like what you're doing going way beyond just datasources. I think this would be very helpful content to share. Do you want to write a blog post that I can refer to? Or do you have something else in mind?

Reply:
Yes, I will write a blog post.